src/common/executor.ts

```diff
@@ -2,25 +2,27 @@
 // Licensed under the MIT license.
 
 "use strict";
-import { ChildProcess, exec, execSync, spawn, SpawnOptions } from "child_process";
+import { ChildProcess, execSync, ExecSyncOptions, spawn, SpawnOptions } from "child_process";
 import * as vscode from "vscode";
+import { Configuration } from "./configuration";
 import { Constants } from "./constants";
 
 export class Executor {
     public static runInTerminal(command: string, terminal: string = Constants.edgeDisplayName): void {
         if (this.terminals[terminal] === undefined) {
-            this.terminals[terminal] = vscode.window.createTerminal(terminal);
+            this.terminals[terminal] = Executor.createTerminal(terminal);
         }
         this.terminals[terminal].show();
         this.terminals[terminal].sendText(command);
     }
 
-    public static exec(command: string) {
-        return exec(command);
-    }
-
     public static execSync(command: string) {
-        return execSync(command, { encoding: "utf8" });
+        const envVars = Executor.getEnvFromConfig();
+        const options: ExecSyncOptions = { encoding: "utf8" };

```

Comment:
Same here: `{ encoding: "utf8", env:  Executor.mergeEnvs(envVars, process.env)}`

Reply:
This will add an key with undefined value. It is different from there is no key.